.github/workflows/ci.yml

```diff
@@ -97,7 +97,8 @@ jobs:
         run: |
           # make sure docker exist
           docker ps
-          docker build  -f Dockerfiles/pip.Dockerfile -t jinaai/jina:test-pip .
+          docker build -f Dockerfiles/pip.Dockerfile -t jinaai/jina:test-pip .
```

Comment:
can we split this in step generating `test-pip` and doing this test?

Reply:
what do you mean here? 1. this line does not change,  2. `test-pip` is not testing, it is a testbed for later tests